deepset_cloud_sdk/workflows/user_facing_docs/pipeline_service_docs.py

```diff
@@ -0,0 +1,55 @@
+"""Documentation strings for the pipeline service module.
```

Comment:
@faymarie how are these docs user facing ? 
AFAIK the markdown generation works by instpecting docstrings etc. Did you check if this works ?

Reply:
I expect the user to see these docstrings when they are using the SDK workflow. They are not used by OpenAPI specs, but when they inspect descriptions, or run into Validation errors or similiar when defining the PublishingConfig (e.g. inputs). It's also user facing, isn't it? To be honest, I just collected them there to have them defined in one place and they can be easier reviewed and maintained. But, good point to check if the markdown generation works in these cases. Not sure, if we would need it anywhere. I'll have a look.